### PR TITLE
finish writing `splitter` function and pass all test cases

### PR DESCRIPTION
**Pull request by jchen9314**

### CODE REVIEW

File: prepackPy/splitter.py [hunk 1]
```diff
@@ -1,33 +1,94 @@
...
+    X_test: numpy ndarray 
+        splitted features for model testing
+    y_test: numpy ndarray
         splitted target for model testing
+        
+    Example
+    -------
+    >>> import numpy as np
+    >>> import pandas as pd
+    >>> from splitter import splitter # to do
```
Comment (smwatts) [inline prepackPy/splitter.py]: I'm a bit confused about the `to do` comment here!

**Review by smwatts — APPROVED** [review]
Just the one comment otherwise LGTM!

File: prepackPy/splitter.py (hunk 1 above)
Comment (jchen9314) [reply prepackPy/splitter.py]: Thanks for pointing that. Aha, my bad. I will fix it.

File: prepackPy/splitter.py [hunk 2]
```diff
@@ -1,33 +1,94 @@
...
+        proportion of the dataset to include in the test split
     seed: integer
-        a random state that will make code reproducible
+        random state that will make code reproducible
 
     Returns
     -------
-    X_train: numpy array
+    X_train: numpy ndarray
         splitted features for model training
```
Comment (apchiodo) [inline prepackPy/splitter.py]: I'm not sure 'splitted' is correct grammar.  Maybe "split features for model training"?

File: prepackPy/splitter.py [hunk 3]
```diff
@@ -1,33 +1,94 @@
...
-    X_test: numpy array 
-        splitted features for model testing
-    y_train: 1d numpy array
+    y_train: numpy ndarray
         splitted target for model training
```
Comment (apchiodo) [inline prepackPy/splitter.py]: Same thing here.

File: prepackPy/splitter.py [hunk 4]
```diff
@@ -1,33 +1,94 @@
...
-    y_test: 1d numpy array
+    X_test: numpy ndarray 
+        splitted features for model testing
```
Comment (apchiodo) [inline prepackPy/splitter.py]: Here.

File: prepackPy/splitter.py [hunk 5]
```diff
@@ -1,33 +1,94 @@
...
+    y_test: numpy ndarray
         splitted target for model testing
```
Comment (apchiodo) [inline prepackPy/splitter.py]: Here.

**Review by apchiodo — APPROVED** [review]

File: prepackPy/splitter.py (hunk 2 above)
Comment (jchen9314) [reply prepackPy/splitter.py]: Ok, I will fix them.

**Review by apchiodo — APPROVED** [review]

**Pull request merged**